src/components/SocialSharingButtons.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+  <div class="social-container">
+    <h1>Share on</h1>
```

Comment:
The text must be translatted (english and french).

In french : "Partager"  
In english : "Share on"

Locales are stored here : src/plugins/translation/

Reply:
> @lgomesp Fine !
> 
> Can you please keep only the following socials :
> 
> * email
> * facebook
> * linkedin
> * reddit
> * sms
> * telegram
> * twitter
> * whatsapp

This is done!